pkg/controllers/job/job_controller_actions.go

```diff
@@ -44,6 +44,10 @@ import (
 
 var calMutex sync.Mutex
 
+const (
+	customLaunch = "volcano.sh/custom-launch"
```

Comment:
BTW, what's the meaning of this annotation? Pls add some comments about that.

Reply:
I want to use this annotation to mark jobs that have task dependencies，or just change the key as “has-task-dependencies”